### PR TITLE
8341055: Open some TextArea awt tests 2

### PR DESCRIPTION
**Pull request by alisenchung**
Second set of textarea awt tests to open during fall 2024 test sprint

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8341055](https://bugs.openjdk.org/browse/JDK-8341055): Open some TextArea awt tests 2 (**Bug** - P4)

### Reviewers
 * [Phil Race](https://openjdk.org/census#prr) (@prrace - **Reviewer**)
 * [Abhishek Kumar](https://openjdk.org/census#abhiscxk) (@kumarabhi006 - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/21436/head:pull/21436` \
`$ git checkout pull/21436`

Update a local copy of the PR: \
`$ git checkout pull/21436` \
`$ git pull https://git.openjdk.org/jdk.git pull/21436/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 21436`

View PR using the GUI difftool: \
`$ git pr show -t 21436`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/21436.diff">https://git.openjdk.org/jdk/pull/21436.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/21436#issuecomment-2403699953)

### CODE REVIEW

File: test/jdk/java/awt/TextArea/TextAreaHScrollbarTest.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+ */
+
+import java.awt.Frame;
+import java.awt.TextArea;
+
+/*
+ * @test
+ * @bug 4648702
+ * @summary TextArea horizontal scrollbar behavior is incorrect
+ * @library /open/test/jdk/java/awt/regtesthelpers
```
Comment (prrace) [inline test/jdk/java/awt/TextArea/TextAreaHScrollbarTest.java:31]: remove leading /open/test

File: test/jdk/java/awt/TextArea/TextAreaLineScrollWrapTest.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+ */
+
+import java.awt.Frame;
+import java.awt.TextArea;
+
+/*
+ * @test
+ * @bug 4776535
+ * @summary Regression: line should not wrap around into multi lines in TextArea.
+ * @library /open/test/jdk/java/awt/regtesthelpers
```
Comment (prrace) [inline test/jdk/java/awt/TextArea/TextAreaLineScrollWrapTest.java:31]: remove leading /open/test

File: test/jdk/java/awt/TextArea/TextAreaScrollbarTest.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+import java.awt.TextArea;
+
+/*
+ * @test
+ * @bug 4158997
+ * @key headful
+ * @summary Make sure that the TextArea has both horizontal and
+ * vertical scrollbars when bad scrollbar arguments are passed
+ * into the constructor.
+ * @library /open/test/jdk/java/awt/regtesthelpers
```
Comment (prrace) [inline test/jdk/java/awt/TextArea/TextAreaScrollbarTest.java:36]: remove leading /open/test

File: test/jdk/java/awt/TextArea/TextScrollTest.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+import java.awt.BorderLayout;
+import java.awt.Frame;
+import java.awt.Panel;
+import java.awt.TextArea;
+
+/*
+ * @test
+ * @bug 4127272
+ * @summary TextArea displays head of text when scrolling horizontal bar.
+ * @library /open/test/jdk/java/awt/regtesthelpers
```
Comment (prrace) [inline test/jdk/java/awt/TextArea/TextScrollTest.java:33]: remove leading /open/test

File: test/jdk/java/awt/TextArea/TextAreaScrollbarTest.java [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+        TextArea t9 = new TextArea(3, 8);
+        t9.setText(both + both + both + both + both + both);
+        add(new Label("Both scrollbars: TextArea(int rows, int columns)"));
+        add(t9);
+
+        TextArea t10 = new TextArea(both + both + both + both + both + both, 3, 8);
+        add(new Label("Both scrollbars: TextArea(text, rows, columns)"));
+        add(t10);
+
+        setSize(600,600);
```
Comment (turbanoff) [inline test/jdk/java/awt/TextArea/TextAreaScrollbarTest.java:113]:
```suggestion
        setSize(600, 600);
```

**Review by prrace — APPROVED** [review]

**Review by kumarabhi006 — APPROVED** [review]

**Pull request closed without merging**